src/lib/twitter_web/views/page_view.ex

```diff
@@ -16,4 +16,26 @@ defmodule TwitterWeb.PageView do
     |> String.split(" ")
     |> hd()
   end
+
+  @doc """
+  Returns true if the tweet is included within the user's favorite posts.
+
+  ## Example
+
+    TwitterWeb.PageView.liked?([],%Twitter.Tweets.Post{})
+    false
+
+    TwitterWeb.PageView.liked?([%Twitter.Tweets.FavoritePosts{}],%Twitter.Tweets.Post{})
+    true
+
+  """
+  @spec liked?(list(Twitter.Tweets.FavoritePosts.t()) | [], Twitter.Tweets.Post.t()) ::
```

Comment:
you should add `alias Twitter.Tweets.FavoritePosts` on the top of this module and fix this like below:
```suggestion
  @spec liked?(list(FavoritePosts.t()), Twitter.Tweets.Post.t()) ::
```

By the way, `list` type includes empty list type. So, you shouldn't add `| []`.

Reply:
Thank you for the comment. Does this apply to other structures within this module and for other modules as well? For example, should I add `alias Twitter.Tweets.Post` in this module and write `liked?(list(FavoritePosts.t()), Post.t())` in the spec instead?

Reply:
I think it is a good idea to keep it DRY as much as possible, and if there are conflicts, you can always rename the alias using `as` syntax, e.g. `alias Twitter.Tweets.FavoritePosts as MostFavoritePosts'